app/models/components/course/achievements_ability_component.rb

```diff
@@ -23,6 +24,12 @@ def allow_staff_manage_achievements
     can :manage, Course::Achievement, course_staff_hash
   end
 
+  def do_not_allow_staff_award_automatically_awarded_achievements
```

Comment:
Currently the convention is `do_not_allow` - but only used in 1 other method (other than this). 

There isn't a disallow in any method names that I can find atm, maybe we can just stick to this?